featuretools/io/sql.py

```diff
@@ -0,0 +1,79 @@
+import pandas as pd
+import sqlalchemy as sa
+
+import featuretools as ft
+
+
+def from_sql(id, connection):
+    loader = EntitySetLoaderFromSQL(connection=connection, id=id)
+    loader.load_all_tables()
+    loader.load_relationships()
+    return loader.es
+
+
+def _find_pk_name(table):
+    column_names = table.columns.keys()
+    pks = [name for name in column_names if table.columns[name].primary_key]
+    if len(pks) > 1:
+        raise RuntimeError('Composite primary key detected')
```

Comment:
a) is there a better idiom than this `if`/`else`?
b) is there a more specific type of error we can raise?